tests/console/validate_autoyast.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: smoke test for autoyast post-installation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use y2logsstep;
+use version_utils 'is_opensuse';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    record_info('INFO', 'Check environment');
+    assert_script_run "env | grep \"SHELL=/bin/bash\"";
+    assert_script_run "env | grep \"HOME=\/root\"";
+    assert_script_run "env | grep \"OSTYPE=linux\"";
+
+    record_info('INFO', 'Check user');
+    assert_script_run "id -u $username |grep 1000";
+
+    if (not is_opensuse()) {
```

Comment:
I though we agreed to flip `if` with `else` to avoid not needed negation, or am I missing something?

Reply:
this has been already solved

Reply:
You mean that you will push your changes? I do not understand what you mean by it's already solved, as I see code as it was before =(

Reply:
ok. i apologize because it was a mess with the rebases and al the pull and push. Anyway it is ok now